### DISCUSSION TITLE
Add option to always show score in multiplayer

### DISCUSSION THREAD

**Discussion by Morodar** [ppy/osu#14140] [Ideas] [answered]
# Summary

Add an option to always show the score in multiplayer.  
Classic allows toggling it using the TAB key.

# osu!lazer
As for now, the score collapses while playing and shows up during breaks.

![Animation5](https://user-images.githubusercontent.com/10113399/128398376-628f0925-498a-4537-92cd-519761a4a2e2.gif)

# osu!stable
In stable however, the score and the combo can be shown all the time.  

![Animation6](https://user-images.githubusercontent.com/10113399/128400893-262ec835-d9b3-4dfd-b8c7-dbaee02e4826.gif)

# Why?

For me, I really enjoy seeing the combo of the opponents giving an extra kick which let me feel the game more intense.  
I'm pretty sure I'm not the only one out there! 😄

**Comment by peppy** [comment, marked as answer]
Yeah we definitely need this back. I'll look into it.

**Reply by frenzibyte** [reply]
Resolved in https://github.com/ppy/osu/pull/14144.

**Reply by Morodar** [reply]
Wow, this was fast. Thanks peppy! :D
